internal/scorecard/plugins/basic_tests.go

```diff
@@ -170,15 +170,16 @@ func (t *WritingIntoCRsHasEffectTest) Run(ctx context.Context) *schelpers.TestRe
 		if !ok {
 			continue
 		}
-		if method == "PUT" || method == "POST" {
+
+		if method == "PUT" || method == "POST" || method == "PATCH" {
```

Comment:
Non-blocking nit: change these to `http.MethodPut`, `http.MethodPost`, etc.

Reply:
fixed, good catch!